src/shared/modules.mjs

```diff
@@ -854,6 +854,167 @@ const generateEventListenResponse = json => {
     return json
 }
 
+const getRefDefinition = (uri = '', moduleJson = {}) => {
+  if (!uri) {
+    throw "getRefDefinition requires a non-null uri parameter"
+  }
+  if (uri.startsWith('#')) {
+    const path = (uri).split('#').pop()
+
+    let definition
+    if (path) {
+     definition = getPathOr(null, path.slice(1).split('/'), moduleJson)
+    }
+    if (!definition) {
+      throw `getRefDefinition: Path '${uri}' not found in ${moduleJson ? (moduleJson.title || moduleJson.info.title) : moduleJson}.`
+      return null
+    }
+    else {
+      return definition
+    }
+  }
+  else {
+    throw `Cannot call getRefDefinition with ${uri}`
+  }
+}
+
+const getAnyOfSchema = (inType, json, anyOfTypes) => {
```

Comment:
Why is `anyOfTypes` a parameter here? I noticed we are just passing in an empty array every time. Maybe it should be local to the `getAnyOfSchema` method body?

Reply:
yes, this added to check the anyOf already defined, that means it is defined for params, and no need to proceed further parsing on this. like that added